protocol-designer/src/pages/CreateNewProtocolWizard/SelectFixtures.tsx

```diff
@@ -12,97 +13,167 @@ import {
-                      without(additionalEquipment, ae)
-                    )
-                  }}
-                  header={t(`${ae}`)}
-                  leftHeaderItem={
-                    <AdditionalEquipmentDiagram additionalEquipment={ae} />
+    <HandleEnter onEnter={handleProceed}>
+      <WizardBody
+        stepNumber={5}
+        header={t('add_fixtures')}
+        subHeader={t('fixtures_replace')}
+        disabled={!hasTrash}
+        goBack={() => {
+          goBack(1)
+        }}
+        proceed={handleProceed}
+      >
+        <Flex marginTop={SPACING.spacing60} flexDirection={DIRECTION_COLUMN}>
+          <Flex flexDirection={DIRECTION_COLUMN}>
+            <StyledText
+              desktopStyle="headingSmallBold"
+              marginBottom={SPACING.spacing12}
+            >
+              {t('which_fixtures')}
+            </StyledText>
+          </Flex>
+          <Flex gridGap={SPACING.spacing4} flexWrap={WRAP}>
+            {filteredAdditionalEquipment.map(equipment => (
+              <EmptySelectorButton
+                disabled={numSlotsAvailable === 0}
+                key={equipment}
+                textAlignment={TYPOGRAPHY.textAlignLeft}
+                size="small"
+                iconName="plus"
+                text={t(`${equipment}`)}
+                onClick={() => {
+                  if (numSlotsAvailable === 0) {
+                    makeSnackbar(t('slot_limit_reached') as string)
+                  } else {
+                    setValue('additionalEquipment', [
+                      ...additionalEquipment,
+                      equipment,
+                    ])
                   }
-                />
-              </ListItem>
+                }}
+              />
             ))}
           </Flex>
+          <Flex marginTop={SPACING.spacing32} flexDirection={DIRECTION_COLUMN}>
+            <StyledText
+              desktopStyle="headingSmallBold"
+              marginBottom={SPACING.spacing12}
+            >
+              {t('fixtures_added')}
+            </StyledText>
+            <Flex flexDirection={DIRECTION_COLUMN} gridGap={SPACING.spacing4}>
+              {filteredDuplicateStagingAreas.map(ae => {
+                const numStagingAreas = filteredAdditionalEquipmentWithoutGripper.filter(
+                  additionalEquipment => additionalEquipment === 'stagingArea'
+                )?.length
+
+                const dropdownProps = {
+                  currentOption: {
+                    name: `${numStagingAreas}`,
+                    value: `${numStagingAreas}`,
+                  },
+                  dropdownType: 'neutral' as DropdownBorder,
+                  filterOptions: getNumOptions(
+                    numSlotsAvailable >= 4
+                      ? 4
+                      : numSlotsAvailable + numStagingAreas - (hasTC ? 1 : 0)
+                  ),
+                  onClick: (value: string) => {
+                    const num = parseInt(value)
```

Comment:
num is a little bit vague
```suggestion
                    const inputNum = parseInt(value)
```